source/puppet/4.1/reference/resources_package_windows.markdown

```diff
@@ -102,6 +102,8 @@ It's a good idea to use the hash notation for file path arguments since they may
 install_options => [ { 'INSTALLDIR' => "${packagedir}" } ]
 {% endhighlight %}
 
+I'm not sure this ^ is necessary as puppet should quote arguments with spaces automatically.
+
```

Comment:
I just went through this with chocolatey https://forge.puppetlabs.com/chocolatey/chocolatey/readme#install-options-with-quotes--spaces